File: 백준/Bronze/2506. 점수계산/점수계산.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt(); 
        sc.nextLine(); 

        int ans = 0;
        int score = 0;
        
        String nums = sc.nextLine(); 
        String[] numberStrings = nums.split(" ");
        
        int[] scores = new int[n];
        
        for (int i = 0; i < n; i++) {
            scores[i] = Integer.parseInt(numberStrings[i]);
        }
        
        for (int i = 0; i < n; i++) {
            if (scores[i] == 1) {
                score += 1;
                ans += score;
            } else {
                score = 0;
            }
        }
        
        System.out.println(ans);
    }
}

File: 백준/Bronze/2506. 점수계산/점수계산.py
import sys
input = sys.stdin.readline

n = int(input())
scores = list(map(int, input().split()))
ans = 0
current = 0
for i in range(n):
    if scores[i]==1:
        current +=1
        ans += current
    else:
        current  = 0
print(ans)